### ISSUE TITLE
Tron testnet address validation is not working

### ISSUE THREAD

**Issue by teke97** [christsim/multicoin-address-validator#92]
I tried to validate the tron address on the testnet, but it doesn't seem to work.
TYxNX8vuWBao85KTBsywq2Lj3h4EsgEEhF seems to be valid for all networks.

https://tronscan.org/#/address/TYxNX8vuWBao85KTBsywq2Lj3h4EsgEEhF
https://shasta.tronscan.org/#/address/TYxNX8vuWBao85KTBsywq2Lj3h4EsgEEhF
https://nile.tronscan.org/#/address/TYxNX8vuWBao85KTBsywq2Lj3h4EsgEEhF

But the code below doesn't work with testnet address validation.

```js
var WAValidator = require('multicoin-address-validator');

var valid = WAValidator.validate('TYxNX8vuWBao85KTBsywq2Lj3h4EsgEEhF', 'tron', 'prod');
if(valid)
      console.log('This is a valid address');
else
      console.log('Address INVALID');

var valid = WAValidator.validate('TYxNX8vuWBao85KTBsywq2Lj3h4EsgEEhF', 'tron', 'testnet');
if(valid)
      console.log('This is a valid address');
else
      console.log('Address INVALID');
      
// This is a valid address
// Address INVALID
```
      
I didn't find any documents related to the differences between the testnet address and the mainnet address.
And both testnets do not recognize the address from the tests.
https://github.com/christsim/multicoin-address-validator/blob/c1ac6827a988a5f28d629e493b06f9b7109f8cf6/test/wallet_address_validator.js#L573

So, my suggestion is to change the address prefix for the testnet as it appears on the mainnet.
https://github.com/teke97/multicoin-address-validator/commit/d31d1aadeef72c4e90d2d25897f41124cf8d269c

**Comment by sc0Vu** [comment]
It seems they compared first byte to 160 on testnet. I couldn't find any documentation of that testnet address, probably tron had deprecated (https://github.com/tronprotocol/tronweb/blob/master/src/utils/crypto.js#L26). You might want to use prod instead.

**Comment by christsim** [comment]
fixed in 0.5.16
